### DISCUSSION TITLE
Custom dashboard tabs only show either icon or title but not both

### DISCUSSION THREAD

**Discussion by codyc1515** [home-assistant/frontend#19815] [Dashboard]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

![image](https://github.com/home-assistant/frontend/assets/50791984/7e5a58c2-26cc-4222-b425-e66bdbacb2f1)

### Describe the behavior you expected

Something like this. Maybe this is expected but maybe not? I would have thought if you did not want a title to show then you should just leave that field empty (it's not mandatory).

![image](https://github.com/home-assistant/frontend/assets/50791984/7ab3f7b1-c0ef-4084-850a-5a123308295d)

### Steps to reproduce the issue

![image](https://github.com/home-assistant/frontend/assets/50791984/dfbf6acb-9b19-412b-a354-d385f311b5dd)

### What version of Home Assistant Core has the issue?

core-2024.2.0b6

**Comment by ildar170975** [comment]
At least the current behaviour is documented:
![image](https://github.com/home-assistant/frontend/assets/71872483/0588f1a6-f145-47ee-ab1f-93d85dfe3550)

But I would be glad if someone implements a "show both icon & title" feature. Currently this may only be done by card-mod theme.

**Comment by codyc1515** [comment]
Interesting. Okay, I will take a look at it.

**Comment by karwosts** [comment]
Going to bump this over to feature requests, as it has always been this way as long as I can remember, and is documented as the expected behavior.

**Comment by elchininet** [comment]
I maintain [this custom HACS plugin](https://github.com/elchininet/keep-texts-in-tabs) to achieve the same. Until this is not officialy supported, it does the same work pretty well.
Regards
